app/assets/stylesheets/core/_type.scss

```diff
@@ -35,11 +37,10 @@ h1, h2, h3, h4, h5, h6 {
   text-rendering: optimizelegibility; // Fix the character spacing for headings
 }
 
-h1 { font-size: 29px; }
-h2 { font-size: 24px; }
-h3 { font-size: 21px; }
-h4 { font-size: 17px; }
-h5 { font-size: 17px; }
+h1 { font-size: 40px; line-height: 47px; }
```

Comment:
you might want to install a whitespace remover on your editor, or if you're using sublime add `trim_trailing_white_space_on_save: true` to your user preferences.

Reply:
ok! i updated ```trim_trailing_white_space_on_save: true``` on my sublime.

however, it seems our preference is adding a space after ```;``` before ```}```. is this true?

```h1 { font-size: 40px; line-height: 47px; }
h2 { font-size: 24px; line-height: 28px; }
h3 { font-size: 20px; line-height: 23px; }
h4 { font-size: 16px; line-height: 19px; }```

Reply:
@dennyluan said what is important is that our formatting is are consistent